docs/software/rpm-development-guide.md

```diff
@@ -249,16 +249,24 @@ In addition, if the repository contains a file called `rpm/<PROJECT>.spec`, it w
 
 ##### Typical workflow when building out of GitHub repos
 
-1. Fork the repository of the package that you would like to build
-1. Create a new branch in your fork
-1. Make, commit, and push changes to your new branch
-1. In your fork, tag the commit that you would like to build
-1. In the `upstream/osg.github.source`, change the repo to point at your fork and tag
-1. Attempt a scratch build
-1. If the build fails, remove the tag and repeat steps 3-6
-1. Submit a PR to merge changes upstream
-1. Tag the final version on the upstream fork
-1. Build the version that will go through the normal software cycle
+This workflow requires write access for the repository of the package that you would like to build in the
+`opensciencegrid` GitHub organization:
+
+1. Clone the repository for the package that you would like to build from the `opensciencegrid` organization
+1. Create a new branch in your local clone named after the version tag you would like to build, e.g. `v1.12.2`.
```

Comment:
it's not a great idea to name your branch exactly the same as a tag (eg `v1.12.2`), because when both exist the ref name is ambiguous.  

`v1.12.2` is typically a tag, but you could get away with `1_12_2` or `v1.12.2-branch` or some other brannch-name scheme.

Reply:
I told him to delete the branch before making the tag.